tmk_core/common/chibios/flash_stm32.c

```diff
@@ -10,19 +10,30 @@
  * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
  * DEALINGS IN THE SOFTWARE.
  *
- * This files are free to use from https://github.com/rogerclarkmelbourne/Arduino_STM32 and 
+ * This files are free to use from https://github.com/rogerclarkmelbourne/Arduino_STM32 and
  * https://github.com/leaflabs/libmaple
  *
  * Modifications for QMK and STM32F303 by Yiancar
  */
 
-#define STM32F303xC
+#if defined(EEPROM_EMU_STM32F303xC)
+    #define STM32F303xC
+    #include "stm32f3xx.h"
+#elif defined(EEPROM_EMU_STM32F103xB)
+    #define STM32F103xB
+    #include "stm32f1xx.h"
+#else
+    #error "not implemented."
+#endif
 
-#include "stm32f3xx.h"
 #include "flash_stm32.h"
 
-#define FLASH_KEY1          ((uint32_t)0x45670123)
-#define FLASH_KEY2          ((uint32_t)0xCDEF89AB)
+#if defined(EEPROM_EMU_STM32F303xC)
+    #define FLASH_KEY1          ((uint32_t)0x45670123)
```

Comment:
As the FLASH keys get loaded in the stm32f3xx.h similarly with what you did for F1xx could you take those defines out and use the ones from the header file?:)

Sorry I should have done that at the time